.golangci.yml

```diff
@@ -23,14 +23,8 @@ linters-settings:
 
 issues:
   exclude-rules:
-    - linters: [staticcheck]
-      text: "\"io/ioutil\" has been deprecated"
     - linters: [errcheck]
       text: "tx\\.Rollback"
-    - linters: [noctx]
-      path: "_test\\.go"
-    - linters: [revive]
-      text: "don't use underscores"
```

Comment:
In general I think these are pretty reasonable exclude rules. We don't need to care about context usage in tests, and we are not obligated to follow some arbitrary set of linter rules. `revive` has lots of rules. Some of them are good, others are questionable, and a few are harmful. We shouldn't be shy about adding excludes here when we find rules that don't work well.

Reply:
I don't disagree but these are rules we already follow in most places in the code base.